### ISSUE TITLE
Previewing content using {{record.field}} displays html in output for text/editor field type

### ISSUE THREAD

**Issue by markehb** [bolt/bolt#7703]
When previewing content using a template that outputs the the property/field of the record object (e.g. {{ record.body }} ), it encodes the html stored in the field value and outputs that.

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 3.6.0
| Install type                   |  tar.gz
| BC Break                     | yes/no
| PHP version                 | 5.6
| Web server                  | Apache

Reproduction
------------
On clean install, create content for page content_type. Add bold/italic/newline etc to body field in editor.
Save and preview.

### Expected result
Body field HTML to be output as:
```
<p>test</p>

<p></p>

<p><strong>test</strong></p>

<p></p>
```

### Actual result
Body field HTML output as:

```
&lt;p&gt;test&lt;/p&gt;

&lt;p&gt;&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;test&lt;/strong&gt;&lt;/p&gt;

&lt;p&gt;&lt;/p&gt;

&lt;p&gt;test&lt;/p&gt;
```

Doesn't occur in v3.5 for what I can see.

Thanks,
Mark

**Comment by bobdenotter** [comment]
Hi @markehb, This is a known issue, and not easy to fix in Bolt 3.x, i'm afraid. It shouldn't show up in the _normal_ view of the page, right?

**Comment by markehb** [comment]
Hi @bobdenotter 

It's fine when viewing normally, just when viewing the preview this occurs. There no plans to resolve this before 4.0?

**Comment by bobdenotter** [comment]
@markehb It's far from trivial to fix, because of historical reasons.. Fixing it, breaks it in different ways.

**Comment by markehb** [comment]
Ok, thanks bob.

**Comment by JarJak** [comment]
html fields should be displayed as {{ record.body|raw }} or with proper escaping method

**Comment by bobdenotter** [comment]
Fixed as a workaround in #7753.
